doc/Command_Index_internal.tex

```diff
@@ -748,7 +740,7 @@ \section{Gregorio\TeX{} Controls}
 \end{argtable}
 
 \macroname{\textbackslash gre@char@cavum}{\#1\#2\#3\#4\#5\#6\#7\#8}{gregoriotex-signs.tex}
-Macro to typeset a "cavum" character.
+Macro to typeset a “cavum” character.
```

Comment:
This is a case where I don't think consistency is all that important.  Both result in the right thing showing up in the pdf when the document is processed by LuaTeX.

I normally use TeX double backquotes/apostrophes because my system didn't handle unicode very well when I first was learning TeX.  However, I now have a XeLaTeX class which I use for school papers and XeLaTeX doesn't handle TeX double apostorphes very well (the actually come out as two single curly quotes instead of one double curly quotes) so I have to turn on my editor's Smart Quotes replacement when using it.  As a result my old habit isn't as strong as it used to be and when I then switch to another document, I sometimes forget to turn the replacements off and will end up with unicode quotes as a result.